### PR TITLE
Warnings2

### PR DESCRIPTION
**Pull request by insertinterestingnamehere**

### CODE REVIEW

**Comment by ddn0** [comment]
the checks are passing yet

File: libgalois/include/galois/DynamicBitset.h [hunk 1]
```diff
@@ -26,16 +26,19 @@
 #ifndef _GALOIS_DYNAMIC_BIT_SET_
 #define _GALOIS_DYNAMIC_BIT_SET_
 
+#include <climits>
```
Comment (ddn0) [inline libgalois/include/galois/DynamicBitset.h:29]: this is my favorite include file

Comment (insertinterestingnamehere) [reply libgalois/include/galois/DynamicBitset.h:29]: Agreed!

**Comment by insertinterestingnamehere** [comment]
Thanks for following through the demo!

**Pull request closed without merging**